Create Jekyll project and render FAQs

til opened the pull request:
As started here https://chaos.social/@ConnyDuck/112304265215565337: convert the static website to a Jekyll project so that there is a build step that can render the FAQ to HTML while still being able to author it in Markdown.

I tried to change as little as possible on the rest of the site for now, only what is necessary to add the automatically rendered FAQ page. The new FAQ page at `/faq/` is styled similar to the privacy policy page. On hover, anchor links to individual headlines are shown and point to e.g. `/faq/#why-is-the-tusky-release-on-f-droid-out-of-date`.

![Screenshot_20240422_101809](https://github.com/tuskyapp/tuskyapp.github.io/assets/11211/4df71742-d3b4-4b8c-ac07-5cfa2993fa75)

The new FAQ page is not linked from the root page yet so that this PR could be merged first to see if the build is working, and publish the new FAQs later.

Let me know where you want to go from here, I'm happy to make more changes but beware I'm not a great designer 😄.

connyduck reviewed (commented):
Wow this is pretty much as I imagined, thank you very much!

Also it was super easy to set up locally.

Some new faqs have been merged recently in the other repo, can you add them here as well please?

Also, we have a [french translation](https://github.com/tuskyapp/faq/blob/main/translated/french_faq.md) of the faqs, can we have them here as well? Maybe /faq/fr or something like that?

til commented:
Happy to hear that!

The english FAQ contents are now updated to the latest version.

I've added the french version under `/fr/faq/` – having the language as two-letter prefix of the URL is pretty standard. This could eventually be extended, if so desired, e.g. with `/fr/privacy-policy/`, `/fr/` for the start page etc.

Tak commented:
Thanks! ❤️
